build.psm1

```diff
@@ -314,7 +314,7 @@ function Start-PSBuild {
     }
```

Comment:
@JamesWTruher, your last commit had 1 failures in `PowerShell-CI-windows`
Remove-Item UnAuthorized Access.Access-denied test for removing a folder
```powershell
Expected exactly 'RemoveItemUnauthorizedAccessError,Microsoft.PowerShell.Commands.RemoveItemCommand', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\FileSystem.Tests.ps1: line 1424
1424:         Get-Content $errorFile | Should -BeExactly 'RemoveItemUnauthorizedAccessError,Microsoft.PowerShell.Commands.RemoveItemCommand'
```